--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -402,12 +402,12 @@ if (WITH_COMPLETION)
             FILES
                 "${PROJECT_SOURCE_DIR}/scripts/bash-completion/pdal"
             DESTINATION
-                "${CMAKE_INSTALL_DATAROOTDIR}/share/bash-completion/completions")
+                "${CMAKE_INSTALL_PREFIX}/share/bash-completion/completions")
     elseif (IS_DIRECTORY /etc/bash_completion.d)
         install(
             FILES
                 "${PROJECT_SOURCE_DIR}/scripts/bash-completion/pdal"
             DESTINATION
-                "${CMAKE_INSTALL_DATAROOTDIR}/etc/bash_completion.d")
+                "${CMAKE_INSTALL_PREFIX}/etc/bash_completion.d")
     endif()
 endif()
