diff -rupN --no-dereference PDAL-2.5.5-src/cmake/macros.cmake PDAL-2.5.5-src-new/cmake/macros.cmake
--- PDAL-2.5.5-src/cmake/macros.cmake	2023-06-13 16:04:01.000000000 +0200
+++ PDAL-2.5.5-src-new/cmake/macros.cmake	2023-06-15 07:42:42.924405782 +0200
@@ -228,6 +228,6 @@ macro(PDAL_ADD_TEST _name)
             "PDAL_DRIVER_PATH=${PROJECT_BINARY_DIR}/bin")
     else()
         set_property(TEST ${_name} PROPERTY ENVIRONMENT
-            "PDAL_DRIVER_PATH=${PROJECT_BINARY_DIR}/lib")
+            "PDAL_DRIVER_PATH=${PROJECT_BINARY_DIR}/lib${LIB_SUFFIX}")
     endif()
 endmacro(PDAL_ADD_TEST)
